src/app/my-donations/my-donations.component.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, Inject, OnInit, PLATFORM_ID} from '@angular/core';
+import {ComponentsModule} from "@biggive/components-angular";
+import {PageMetaService} from "../page-meta.service";
+import {IdentityService} from "../identity.service";
+import {DonationService} from "../donation.service";
+import {Person} from "../person.model";
+import {Router} from "@angular/router";
+import {Observable} from "rxjs";
+import {Donation} from "../donation.model";
+import {AsyncPipe, DatePipe} from "@angular/common";
+import {ExactCurrencyPipe} from "../exact-currency.pipe";
+import {MatProgressSpinner} from "@angular/material/progress-spinner";
+
+@Component({
+  selector: 'app-my-donations',
+  standalone: true,
+  imports: [
+    ComponentsModule,
```

Comment:
Maybe worth swapping for `...allChildComponentImports`? I forget the details but I think things like router link behaviour were subtly different when some pages were missing that import – probably good to do it the same throughout the app.

Reply:
Right, done in d1de1cd9 . I had just used the angular command to create the my-donations component, didn't remember about allChildComponentImports